src/components/List.js

```diff
@@ -1,5 +1,27 @@
+import { useState, useEffect } from 'react';
+//firebase
+import { db } from '../lib/firebase';
+import { onSnapshot, collection, query, where } from 'firebase/firestore';
+
 const List = () => {
-  return <div>List</div>;
+  const [datas, setData] = useState([]);
+  useEffect(() => {
+    const ListRef = collection(db, 'List1');
+    const q = query(ListRef, where('token', '==', 'reda and tobi'));
+    const unsb = onSnapshot(q, ListRef, (snapshot) => {
```

Comment:
Maybe these two variables `q` and `unsb` can have more descriptive names so they're easier to read and understand :slightly_smiling_face: ?